torch/_prims/nvfuser_executor.py

```diff
@@ -268,6 +268,29 @@ def __call__(self, *args):
         )
 
 
+# A set of operators that are supported by nvFuser
+# but should not form a fusion group solely on their own
+# _non_compute_ops = {
```

Comment:
nitpick: commented code should be removed.

Reply:
It's an example, but it should have been marked so more clearly 😄